app/features/navbar/components/HelpButton.js

```diff
@@ -101,10 +77,10 @@ export default class HelpButton extends Component< *, State> {
                 position = 'right bottom'
                 trigger = { <HelpIcon /> }>
                 <Group heading = 'Help'>
-                    <Item onActivate = { this._onTermsOptionClick }>
+                    <Item onActivate = { this._onTerms }>
```

Comment:
I guess it should be at least `onTermsClick`?

Reply:
I went with the shorter version, WDYT?

Reply:
It just doesn't feel like a listener function anymore. 😅

Reply:
Yeah, you’re right. I’ll fix that!